src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloads.Fixer.cs

```diff
@@ -33,6 +34,9 @@ public abstract class PreferStreamAsyncMemoryOverloadsFixer : CodeFixProvider
         // Checks if the argument in the specified index has a name. If it doesn't, returns that arguments. If it does, then looks for the argument using the specified name, and returns it, or null if not found.
         protected abstract IArgumentOperation? GetArgumentByPositionOrName(ImmutableArray<IArgumentOperation> args, int index, string name, out bool isNamed);
 
+        // 
```

Comment:
Forgot to leave the comment?

Reply:
Huh weird, I thought I added a comment on top of this method. Thanks for catching.